src/main/scanCache/scanCacheObject.ts

```diff
@@ -1,21 +1,41 @@
 import { IArtifact } from 'jfrog-client-js';
+import { IIssueKey } from '../types/issueKey';
+import { INodeInfo } from '../types/nodeInfo';
+import { Severity } from '../types/severity';
+import { Translators } from '../utils/translators';
 
 export class ScanCacheObject {
```

Comment:
Let's a description for this class and explain how it us used by the scan cache manager.